### PR TITLE
AC-460: Changed MaterialTabHost to TabLayout and added the Tabs to be…

### PR DESCRIPTION
**Pull request by misikirmehari**
… Scrollable

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/PatientDashboardPagerAdapter.java [hunk 1]
```diff
@@ -41,15 +46,24 @@
     private static final int VITALS_TAB_POS = 3;
     private static final int CHARTS_TAB_POS = 4;
 
+
+
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/PatientDashboardPagerAdapter.java]: What are these newlines for?

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/PatientDashboardPagerAdapter.java [hunk 2]
```diff
@@ -41,15 +46,24 @@
...
+
     PatientDashboardPagerAdapter(FragmentManager fm, String id) {
         super(fm);
         this.mPatientId = id;
     }
 
+
+
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/PatientDashboardPagerAdapter.java]: I think there's no need for these newlines.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/PatientDashboardPagerAdapter.java [hunk 3]
```diff
@@ -78,6 +92,26 @@ public Fragment getItem(int i) {
...
+            case VISITS_TAB_POS:
+                return tabNames.get(i);
+            case VITALS_TAB_POS:
+                return tabNames.get(i);
+            case CHARTS_TAB_POS:
+                return tabNames.get(i);
+            default:
+                return null;
+        }
+
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/PatientDashboardPagerAdapter.java]: You can simplify this to something like
```java
case DETAILS_TAB_POS:
case DIAGNOSIS_TAB_POS:
case VISITS_TAB_POS:
case VITALS_TAB_POS:
case CHARTS_TAB_POS:
   return tabNames.get(i);
default:
   return null;
```

File: openmrs-client/src/main/res/layout/activity_patient_dashboard.xml [hunk 4]
```diff
@@ -19,16 +19,21 @@
     android:orientation="vertical">
 
     <android.support.design.widget.AppBarLayout
-        android:layout_width="match_parent"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:elevation="0dp">
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/layout/activity_patient_dashboard.xml]: Please indent these to be consistent.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/PatientDashboardPagerAdapter.java [hunk 5]
```diff
@@ -78,6 +92,26 @@ public Fragment getItem(int i) {
...
+                return tabNames.get(i);
+            case VISITS_TAB_POS:
+                return tabNames.get(i);
+            case VITALS_TAB_POS:
+                return tabNames.get(i);
+            case CHARTS_TAB_POS:
+                return tabNames.get(i);
+            default:
+                return null;
+        }
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/PatientDashboardPagerAdapter.java]: You can simplify this to something like
```java
case DETAILS_TAB_POS:
case DIAGNOSIS_TAB_POS:
case VISITS_TAB_POS:
case VITALS_TAB_POS:
case CHARTS_TAB_POS:
   return tabNames.get(i);
default:
   return null;
```

File: openmrs-client/src/main/res/layout/activity_patient_dashboard.xml [hunk 6]
```diff
@@ -19,16 +19,21 @@
...
+
+    <android.support.design.widget.TabLayout
         android:layout_height="wrap_content"
-        android:elevation="0dp">
+        android:layout_width="match_parent"
+        android:id="@+id/tabhost"
+        app:colorTabIndicator="#fff"
+        app:tabTextColor="@color/white"
+        >
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/layout/activity_patient_dashboard.xml]: Put `>` above, next to the last attribute, like other views here.

File: openmrs-client/src/main/res/layout/activity_patient_dashboard.xml (hunk 4 above)
Comment (f4ww4z) [reply openmrs-client/src/main/res/layout/activity_patient_dashboard.xml]: You can also go to `Code` -> `Reformat Code` (if you're using Android Studio) for this file.

**Comment by dkayiwa** [comment]
@misikirmehari are you still working on this?

**Comment by dkayiwa** [comment]
@misikirmehari how is this going?

**Comment by dkayiwa** [comment]
@misikirmehari ping me to reopen when you resume working on this.

**Pull request closed without merging**